src/clj/battlebots/arena/generation.clj

```diff
@@ -11,7 +11,7 @@
 (defn- pos-open
   "returns true of false depending if a given coodinate in a given arena is open"
   [[x y] arena]
-  (= (:open arena-key) (get-in arena [x y])))
+  (= (:open arena-key) (get-in arena [y x])))
```

Comment:
Good catch! I missed this when normalizing all the arena logic. I should have mentioned something to you earlier about this... I realized after the hackathon that there were two interpretations on how the arena was structured. I may swap out the `get-in` function with the `get-item` function under `arena.utils`. I want to take a look around and see if were doing this anywhere else.